src/registrar/services/dns_host_service.py

```diff
@@ -153,8 +172,9 @@ def register_nameservers(self, domain_name, nameservers):
     def save_db_account(self, vendor_account_data):
         result = vendor_account_data["result"]
         x_account_id = result["id"]
-        dns_vendor = DnsVendor.objects.get(name=DnsVendor.CF)
+        dns_vendor = DnsVendor.objects.get(name=CURRENT_DNS_VENDOR)
```

Comment:
I realized we should use a const to refer to the current vendor so that we can easily change that when we switch.